public/app/features/api-keys/APIKeysMigratedCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, LinkButton, useStyles2 } from '@grafana/ui';
+
+interface Props {
+  onHideApiKeys: () => void;
+}
+
+export const APIKeysMigratedCard = ({ onHideApiKeys }: Props): JSX.Element => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Alert title="API keys were migrated to Service accounts. This tab is deprecated." severity="info">
+      <div className={styles.text}>
+        We have upgraded your API keys into more powerful Service accounts and tokens. All your keys are safe and
+        working - you will find them inside respective service accounts. Keys are now called tokens.
```

Comment:
```suggestion
        working - you will find them inside respective service accounts. Keys are now called tokens. For more information about the benefits of service accounts see <a href="https://grafana.com/docs/grafana/latest/administration/service-accounts/about-service-accounts/#service-accounts-benefits">
```
nit:
A link to the benefits of service accounts might be goo.